src/main/java/com/example/tournamentMaker/auth/AuthenticationService.java

```diff
@@ -1,2 +1,64 @@
-package com.example.tournamentMaker.auth;public class AuthenticationService {
+package com.example.tournamentMaker.auth;
+
+import com.example.tournamentMaker.address.Address;
+import com.example.tournamentMaker.config.JwtService;
+import com.example.tournamentMaker.token.Token;
+import com.example.tournamentMaker.token.TokenRepository;
+import com.example.tournamentMaker.token.TokenType;
+import com.example.tournamentMaker.user.Role;
+import com.example.tournamentMaker.user.User;
+import com.example.tournamentMaker.user.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class AuthenticationService {
+    private final UserRepository repository;
+    private final TokenRepository tokenRepository;
+    private final PasswordEncoder passwordEncoder;
+    private final JwtService jwtService;
+    private final AuthenticationManager authenticationManager;
+
+    public AuthenticationResponse register(RegisterRequest request) {
+        User user = new User(request.getFirstName(), request.getLastName(), new Address(request.getCity(),
```

Comment:
z tego adresu to bym pewnie builder zrobil

Reply:
Dlaczego? przecież ma raptem 2 pola